app/test/service.spec.ts

```diff
@@ -0,0 +1,28 @@
+import oasis from '@oasislabs/client';
+
+jest.setTimeout(20000);
+
+describe('Test', () => {
```

Comment:
I think this is a great start, and can we add some other tests that execute transactions for the counter as well?